CHANGES.txt

```diff
@@ -1,3 +1,6 @@
+7.1.0 (Nov XX, 2021)
+ - Added `keyHashTags` option for cluster to select one against several passed.
```

Comment:
Let's make it something more explanatory, like:
- Added a new option to use when running Redis in cluster mode called `keyHashTags` which receives a list of hashtags from which the SDK will randomly pick one to use on the generated instance. 

feel free to tweak as needed though!